seshdash/tests/test_status_icons.py

```diff
@@ -0,0 +1,64 @@
+from django.test import TestCase, Client
+from django.test.utils import override_settings
+#models
+from seshdash.models import Site_Weather_Data,VRM_Account,Sesh_Site,Status_Rule,BoM_Data_Point
+
+from geoposition import Geoposition
+from django.conf import settings
+#utils
+from django.utils import timezone
+from seshdash.utils.model_tools import get_quick_status
+from django.contrib.auth.models import User
+from seshdash.data.db.influx import Influx
+
+class StatusTestCase(TestCase):
+    @override_settings(DEBUG=True)
+    def setUp(self):
+
+        self.db_name = 'test_db'
```

Comment:
Your code is wrong see below, you're not creating the database,

`
        self._influx_db_name = 'test_db'
        self.i = Influx(database=self._influx_db_name)

        try:
            self.i.create_database(self._influx_db_name)
            #Generate random data  points for 24h
        except:
           self.i.delete_database(self._influx_db_name)
           sleep(1)
           self.i.create_database(self._influx_db_name)
           pass
`